extras/free-books.md

```diff
@@ -87,6 +87,9 @@ Name | Author(s)
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author(s) 
+:-- | :--: 
+[Linear Algebra](https://www.amazon.com/Linear-Algebra-2nd-Kenneth-Hoffman/dp/0135367972) | Kenneth M. Hoffman
```

Comment:
Is that a legitimate source of the book? Might be better to include the paid version on the paid-books.md unless the author/publisher has provided an official free version. Thanks!